test/stateless.test.js

```diff
@@ -38,8 +34,8 @@ describe("stateless component with propTypes", () => {
     })
 
     test("render test correct", async () => {
-        await asyncReactDOMRender(<StatelessCompObserver testProp="hello world" />, testRoot)
-        expect(testRoot.querySelector("div").innerHTML).toBe("result: hello world")
+        const { container } = render(<StatelessCompObserver testProp="hello world" />)
```

Comment:
Perhaps we can catch the drift and start calling these correctly a function components. Even rename a whole test file.

Reply:
It has nothing to do with `react-testing-library`. I can investigate it in the separate PR.